tox.ini

```diff
@@ -12,25 +12,26 @@ envdir =
     pypy3: {toxworkdir}/pypy3
     py: {toxworkdir}/py
 commands =
-    python housekeep.py setup
-    nocov: python -m nose2 -v {posargs}
-    cov: python -m coverage run -m nose2
-    cov: python -m coverage combine
-    cov: python -m coverage html --directory=htmlcov/{envname}
-    cov: python -m coverage report -m --fail-under=100
-    cov: python -m coverage xml
-    diffcov: diff-cover coverage.xml --html-report diffcov.html
-    diffcov: diff-cover coverage.xml --fail-under=100
-    profile: python -m nose2 --profile {posargs}
+    python housekeep.py prep
+    nocov: pytest --verbose -p no:cov {posargs}
```

Comment:
`-p no:cov` is needed because `pytest-cov` insists on initializing itself even if `--cov` is not specified.

This causes breakage with `pypy3-nocov` testenv on FreeBSD, as `coverage.py` demands usage of `_sqlite3_cffi` which is not available on FreeBSD's pypy3.